resume/resume-2.html

```diff
@@ -0,0 +1,118 @@
+					<ul id="cook">
+						<li>Crafting meals to order</li>
+						<li>Maintaining Equipment for both appearance and performance</li>
+						<li>Exploring flavor combinations like a rogue alchemist</li>
+					</ul>
+				</li>
+				<li id="airWaves">Radio<br>
+					<ul id="proder">
+						<li>Production Assistant for local call-in talk show on NPR affliate in Columbus Ohio</li>
+						<li>Freelance Producer for NPR affliate in Dayton Ohio</li>
+					</ul>
+				</li>
+				<li id="liluns">Teaching<br>
+					<ul id="teach">
+						<li>Assisting in creating an environment of learning for 2-4 year olds in a Montessori setting</li>
+					</ul>	
+				</li>
+				<li id="growem">Farming<br>
+					<ul id="vegs">
+						<li>Feeding chickens and collecting eggs</li>
+						<li>Cultivating a wide variety of vegetables over a three acre plot</li>
+						<li>Feeding and caring for pigs</li>
+						<li>Packaging produce for 45 local families weekly</li>
+					</ul>
+				</li>
+			</ul>
+		</div>
+		<h2 class="collapsible">Volunteer Experience</h2>
+		<div class="content">
+			<ul id="volt">
+				<li id="artWork">Pan-Eros<br>
+					<ul id="eros">
+						<li>Managing Security team for internationally renowned Art Festival</li>
+						<li>Cataloging Foundation Art</li>
+						<li>Training and tasking Volunteers</li>
+					</ul>
+					<li id="olymps">Special Olympics<br>
+						<ul id="coach">
+							<li>Coaching: Basketball Individual Skills Training, Swimming, and Softball</li>
+							<li>Participating as a Unified Partner: Swimming, Softball</li>
+							<li>Annoucing: Basketball</li>
+						</ul>
+					</li>
+			</ul>
+		</div>
+		<h2 class="collapsible">References</h2>
+		<div class="content">
+			<ul id="refs">
+				<li id="refsSophia">Sophia Iannicelli<br><p id="frankie">Director, <a href="http//:www.pan-eros.org">Pan-Eros Foundation</a>, <a href="mailto:director@seattleerotic.org">director@seattleerotic.org</a></p></li>
+				<li id="refsMike">Mike Thompson<br><p id="toms">News Director, <a href="https//:www.wosu.org">WOSU Public Media</a>, <a href="mailto:mike.thompson@wosu.org">mike.thompson@wosu.org</a></p></li>
+				<li id="refsChefs">Andre Decker/Colleen Carrol<br><p id="chiefs">Andre is the Head Chef for Daniel's Broiler Downtown. <a href="mailto:adecker@schwartzbros.com">adecker@schwartzbros.com</a><br>Colleen is the Head Chef for Daniel's Broiler Leschi. <a href="mailto:ccarrol@schwartzbros.com">ccarrol@schwartzbros.com</a></p></li>
+			</ul>
+		</div>
+		<h2 class="collapsible">Education</h2>
+		<div class="content">
+			<ul id="knows">
+				<li id="learn">Ohio State<br><p id="deg">In December of 2007, I graduated with a B.A. in English with a minor in theatre</p></li>
+				<li id="cud">Khan Academy<br><p id="KHAN">I spent a week learning how to code so I could do what you see here!</p></li>
+			</ul>
+		</div>
+		<script>
+		var coll = document.getElementsByClassName("collapsible");
+		var i;
+
+		for (i = 0; i < coll.length; i++) {
```

Comment:
this whole bit is cool and snagged from here: https://www.w3schools.com/howto/howto_js_collapsible.asp

the `getElementsByClassName` grabs all those collapsible sections and then iterates through them and adds event listeners to change the styling when a section is clicked.